api/test/nostd/unique_ptr_test.cc

```diff
@@ -108,8 +108,7 @@ TEST(UniquePtrTest, PointerOperators)
   unique_ptr<int> ptr1{value};
 
   EXPECT_EQ(&*ptr1, value);
-  EXPECT_EQ(
-      unique_ptr<B> {}->f(), 123);
+  EXPECT_EQ(nullptr, (unique_ptr<B> {}).get());
```

Comment:
I think this test should rather properly initialize a `unique_ptr<B>` and test calling `f`. Like:
```c++
unique_ptr<int> ptr2{new B};
EXPECT_EQ(123, ptr2->f());
```

Reply:
Done here and also in the shared_ptr test.